docs/index.rst

```diff
@@ -0,0 +1,152 @@
+.. toctree::
+   :hidden:
+
+   quickstart-mdldev
+
+.. toctree::
+   :caption: Introduction to Determined
+   :hidden:
+
+   Features <intro/features/list>
+   Concepts <intro/concepts/overview>
+   System Architecture <intro/architecture/overview>
+
+.. toctree::
+   :caption: Determined Interfaces
+   :hidden:
+
+   WebUI Interface <interfaces/webui-if>
+   Commands and Shells <interfaces/commands-and-shells>
+   Python API <interfaces/python-api>
```

Comment:
Can we refer to this entity as the "Python SDK"?  Which is distinguished from all other python-related APIs because it's the only entity in python for interacting with the cluster from outside of jobs.  "Python SDK" is what we have normally called it in the past.